### ISSUE TITLE
Ability to skip over files

### ISSUE THREAD

**Issue by kylemeier** [DennisBecker/grunt-sass-globbing#19]
I love this plugin!

However, for reasons similar to https://github.com/DennisBecker/grunt-sass-globbing/issues/15, it would be nice to have the ability to skip over files so we can add them separately in the order we need. I believe this is a better solution than changing the import order as it wouldn't mess with anyone's expectations on how the plugin works.

Many other grunt tasks offer this functionality with a syntax like:
```javascript
sass_globbing: {
 'app/assets/css/_importMap.scss': ['app/modules/**/*.scss', '!app/modules/core/utilities/utilities.scss']
}
```

If it helps here are some that work that way: 
https://github.com/gruntjs/grunt-contrib-copy
https://github.com/gruntjs/grunt-contrib-jshint
https://github.com/gruntjs/grunt-contrib-clean

Thanks!

**Comment by DennisBecker** [comment]
Seems a good and viable option.

My only problem is that I'm really busy at the moment but this will be added!

**Comment by jtwalters** [comment]
See #23

**Comment by DennisBecker** [comment]
fixed in #23
